lib/bolt_server/transport_app.rb

```diff
@@ -479,7 +479,7 @@ def make_winrm_target(target_hash)
         'uri' => target_hash['hostname'],
         'config' => {
           'transport' => 'winrm',
-          'winrm' => opts
+          'winrm' => opts.slice(*Bolt::Config::Transport::WinRM.options)
```

Comment:
What does this do? The `slice` I mean.

Reply:
It's equivalent to `select-keys` in clojure. It returns the subset of the hash that corresponds to the keys in the array (ignoring keys that aren't present in the hash).

Reply:
It's also the best method ever created.